tostring method and gematria

philosp opened the pull request:
Hello,
I have used convertdate in a Reddit bot to help translators convert between different calendars.
I've noticed in the Armenian calendar that you have a tostring method that is missing from other calendars, and which would be helpful to me. 
So I've created this branch and PR in order to include the tostring method to other calendars.
In the hebrew calendar, the method includes a number-to-gematria conversion. If this seems off-topic to you, I can revert it and keep the numbers as-is.
I have included the months of the Persian calendar, [as per Wikipedia](https://en.wikipedia.org/wiki/Solar_Hijri_calendar#Month_names).
Thank you.

fitnr commented:
Thanks for the contribution! Adding more of these formatting functions is a great idea. Can you rename them `format()`? It's not an ideal name, but it's the more common one in the library, so we might as well be consistent.

For the months in the `hebrew` module, can you please convert the dictionary to a list? You'll have to adjust the indexing in the function, but `list`s express a linear sequence in a way that `dict`s don't.

 I don't think it's a necessary to add the `gematria` conversion function, I would like to keep the requirements for this package to an absolute minimum.

philosp commented:
Thanks for your reply. 
I have made changes on the three suggestions you've made above:
- renamed the `tostring()` methods to `format()` (In `armenian.py` I've kept the deprecated `tostring()` method as a pure call to `format()`, since it was already in place before this)
- reworked the hebrew months `dict` as a `list`
- removed the `int-to-gematria()` calls in `hebrew.py` and its dependency
